lib/Symbols.js

```diff
@@ -12,7 +13,7 @@ Symbols.getSymbolsInEquation = function(equation) {
 
 // return the set of symbols in the expression tree
 Symbols.getSymbolsInExpression = function(expression) {
-  const symbolNodes = expression.filter(node => node.isSymbolNode); // all the symbol nodes
+  const symbolNodes = filter(expression, Node.Type.isSymbol); // all the symbol nodes
```

Comment:
this should be called `findAll` or something more descriptive - and maybe it'd be better living in semantic-math as a function on the node?